docs/spark.rst

```diff
@@ -290,5 +290,29 @@ Environment knobs
 -----------------
 * ``HOROVOD_SPARK_START_TIMEOUT`` - sets the default timeout for Spark tasks to spawn, register, and start running the code.  If executors for Spark tasks are scheduled on-demand and can take a long time to start, it may be useful to increase this timeout on a system level.
 
+Horovod in Spark on Databricks
+------------------------------
+Horovod in Spark in supported on
+`Databricks Runtime 7.4 ML <https://docs.databricks.com/release-notes/runtime/releases.html>`__
+and above. To run Horovod in Spark on Databricks, use `DBFSLocalStore` as the store object:
+
+.. code-block:: python
+
+    store = DBFSLocalStore(prefix_path='/dbfs/...')
+
+The `DBFSLocalStore` uses `Databricks File System (DBFS) local file APIs <https://docs.databricks.com/data/databricks-file-system.html#local-file-apis>`__
+as a store of intermediate data and training artifacts.
+
+Databricks preconfigures GPU-aware scheduling for you on GPU clusters. See the
```

Comment:
I don't think Databricks setup is any different from a Spark cluster with GPU scheduling enabled. We can just say Databricks pre-configures GPU-aware scheduling on Databricks Runtime 7.0 ML GPU and above, and link to GPU scheduling instructions.